tracer/src/Datadog.Trace/IAST/Aspects/System/UriBuilderAspect.cs

```diff
@@ -0,0 +1,197 @@
+// <copyright file="UriBuilderAspect.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Iast.Dataflow;
+using Datadog.Trace.Iast.Propagation;
+
+#nullable enable
+
+namespace Hdiv.AST.Aspects.System
```

Comment:
Probably want to change this namespace :)

Reply:
Yes, totally right...